README.md

```diff
@@ -41,28 +41,61 @@ This subgraph is deployed under `/subgraphs/name/oceanprotocol/ocean-subgraph/`
 
 ```graphql
 {
-  pools(orderBy: oceanReserve, orderDirection: desc) {
-    consumePrice
-    datatokenReserve
-    oceanReserve
-    spotPrice
-    swapFee
-    transactionCount
+  pools(orderBy: baseTokenLiquidity, orderDirection: desc) {
+    id
+    datatoken {
+      address
+    }
+    baseToken {
+      symbol
+    }
+    baseTokenLiquidity
+    datatokenLiquidity
   }
 }
 ```
 
-**All datatokens**
+**Pools with the highest liquidity**
 
 ```graphql
 {
-  datatokens(orderBy: createTime, orderDirection: desc) {
-    address
+  pools(where: {datatokenLiquidity_gte: 1}, orderBy: baseTokenLiquidity, orderDirection: desc, first: 15) {
+    id
+    datatoken {
+      address
+    }
+    baseToken {
+      symbol
+    }
+    baseTokenLiquidity
+    datatokenLiquidity
+  }
+}
+```
+
+**All Data NFTs**
+
+```graphql
+{
+  nfts(orderBy: createdTimestamp, orderDirection: desc){
```

Comment:
these will not return "all" just the first 100 , you can return 1000 if you explicitly add 'first:1000' . 1000 is the max items returned by subgraph.

Reply:
Cool, I'll update that